src/store/resourcesSlice.ts

```diff
@@ -30,6 +30,16 @@ export type ArticleData = {
   preview: string;
 };
 
+export type VideoArticle = VideoData | ArticleData;
+
+export const videoOrArticle = (tbd: VideoArticle): tbd is VideoData => {
```

Comment:
Maybe change this to `isVideo` for clarity? At first I thought this was meant to return true if the object was either a video or article

Reply:
good catch! will fix rn